### PR TITLE
Make whole row clickable to open TaskExecutionDetails panel

### PR DESCRIPTION
**Pull request by eugenejahn**
# TL;DR
make the row clickable and will open the sidebar

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixeshttps://github.com/flyteorg/flyteconsole/issues/398

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by eugenejahn** [comment]
Goal
...
Ensure that click on a ViewLogs still leads to logs and doesn't open the side panel

---
However, I think the current logic (before the PR) for ViewLogs leads to opening the side panel. not sure is a bug or not.

**Comment by anrusina** [comment]
> However, I think the current logic (before the PR) for ViewLogs leads to opening the side panel. not sure is a bug or not.

Thank you for pointing it out, we will cover it with some other changes. As I don't think we can really open the logs here anyway, as for some tasks there could be multiple logs.

File: packages/zapp/console/src/components/Executions/Tables/utils.ts [hunk 1]
```diff
@@ -4,3 +4,8 @@ import { nameColumnLeftMarginGridWidth } from './styles';
 export function calculateNodeExecutionRowLeftSpacing(level: number, spacing: Spacing) {
   return spacing(nameColumnLeftMarginGridWidth + 3 * level);
 }
+
+export function selectExecution(state, nodeExecution) {
+  // use null in case if there is no execution provied - to close panel
```
Comment (anrusina) [inline packages/zapp/console/src/components/Executions/Tables/utils.ts:9]: nit: provided  (missing one d)

File: packages/zapp/console/src/components/Executions/Tables/utils.ts [hunk 2]
```diff
@@ -4,3 +4,8 @@ import { nameColumnLeftMarginGridWidth } from './styles';
...
+  state.setSelectedExecution(nodeExecution?.id ?? null);
+}
```
Comment (anrusina) [inline packages/zapp/console/src/components/Executions/Tables/utils.ts:11]: I'm not sure it makes sense to add a helper function for such type of call. I think it would be cleaner to just call state.setSelectedExecution(nodeExecution?.id ?? null) in place. Comment about why null is used could leave near `setSelectedExecution` definition if you are trying to avoid it's duplication.

Anyway, if you want to stick with a helper function -> please provide parameters types you are expecting, as:
```
export function selectExecution(state: NodeExecutionsTableState, nodeExecution: NodeExecution) { 
...
}
```

**Review by anrusina — COMMENTED** [review]
The build check is failing as expected.
Tests check is already properly connected, so you would ensure that that one is green. Looks like with your changes `NodeExecutionsTable` test is failing, to test only it please run tests from console directory as:
```
narusina@nastyas-mbp flyteconsole % cd packages/zapp/console
narusina@nastyas-mbp console % yarn test NodeExecutionsTable
```

File: packages/zapp/console/src/components/Executions/Tables/utils.ts (hunk 2 above)
Comment (eugenejahn) [reply packages/zapp/console/src/components/Executions/Tables/utils.ts:11]: agree with that since the logic is pretty simple.

**Review by anrusina — APPROVED** [review]
Nice work,
### 🎉
